Modelica/Resources/C-Sources/ModelicaStandardTables.c

```diff
@@ -4213,6 +4219,8 @@ double ModelicaStandardTables_CombiTable2D_getDerValue(void* _tableID, double u1
         const double u1Max = TABLE_COL0(nRow - 1);
         const double u2Min = TABLE_ROW0(1);
         const double u2Max = TABLE_ROW0(nCol - 1);
+        const double du1 = der_u1 > 0 ? u1*DBL_EPSILON : -u1*DBL_EPSILON;
```

Comment:
Can we also use `_EPSILON` (as defined in the module) instead of `DBL_EPSILON`?

Reply:
I would want to avoid having too large epsilon, I was also considering redesigning findIndex to have an extra argument to break ties to avoid it - to completely avoid the need for any epsilon.

Reply:
We can adapt `_EPSILON` if needed.

Reply:
I changed it to avoid epsilon altogether for this; as I found it more clearly showed the intent.

Reply:
Ah, right -> `findLess`.